### PR TITLE
[MNG-7819] Add IT that excercise the bug

### PR DESCRIPTION
**Pull request by cstamas**
Template/reproducer: https://gist.github.com/cstamas/3f15775bd4785a5f4b157399f52af6fc

While local repo is empty.

---

https://issues.apache.org/jira/browse/MNG-7819

### CODE REVIEW

**Comment by cstamas** [comment]
Adding this IT should be safe: it will fail with 3.9.2 and alpha-5, but it also excludes, and starting with 3.9.3 it should run

File: core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7819FileLockingWithSnapshotsTest.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+class MavenITmng7819FileLockingWithSnapshotsTest extends AbstractMavenIntegrationTestCase {
+
+    private Server server;
+
+    private int port;
+
+    protected MavenITmng7819FileLockingWithSnapshotsTest() {
+        // broken: maven 3.9.2 and 4.0.0-alpha-5
+        super("[3.9.0,3.9.2),(3.9.2,],[4.0.0-alpha-6,)");
```
Comment (cstamas) [inline core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7819FileLockingWithSnapshotsTest.java:45]: [3.9.0,3.9.2),(3.9.2,4.0.0-alpha-5),[4.0.0-alpha-6,) instead?

Comment (cstamas) [reply core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7819FileLockingWithSnapshotsTest.java:45]: bah, no, would skip 3.9.3...

**Review by gnodet — COMMENTED** [review]
Please fix the range to include `3.9.3`, otherwise fine.

**Pull request merged**